galileo/build.gradle

```diff
@@ -87,6 +87,12 @@ dependencies {
     implementation 'android.arch.lifecycle:extensions:1.1.1'
 
     implementation 'com.squareup:seismic:1.0.2'
+
+    implementation "io.arrow-kt:arrow-core:0.8.0"
```

Comment:
with arrow hemos topado, what we use it for?

Reply:
I have added it just for testing Arrow! We could use it to improve our functional programming learnings :smile: